### PR TITLE
Flip the tx history chart #1456

### PR DESCRIPTION
**Pull request by liaojikunwork**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```
Screen Shot 2020-08-05 at 12 39 52 PM

### CODE REVIEW

File: src/shared/home/stats/map-card.tsx [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const MapCard = (): JSX.Element => {
           return null;
         }
         const mapdata = Object.keys(data).map((name, i) => ({
-          name: `Day ${i + 1}`,
+          name: `Day ${days[i]}`,
```
Comment (raullenchai) [inline src/shared/home/stats/map-card.tsx:117]: Can we change to date? 

Example:
Screen Shot 2020-08-05 at 9 41 05 AM

Comment (liaojikunwork) [reply src/shared/home/stats/map-card.tsx:117]: Possible

Comment (raullenchai) [reply src/shared/home/stats/map-card.tsx:117]: let's do it then

Comment (liaojikunwork) [reply src/shared/home/stats/map-card.tsx:117]: Should the date based on UTC-timezone or UTC-7?

**Comment by liaojikunwork** [comment]
Screen Shot 2020-08-05 at 1 09 52 PM

@raullenchai , does it look good?

**Pull request merged**